package/MDAnalysis/analysis/rms.py

```diff
@@ -528,8 +528,7 @@ def _single_frame(self):
             self._ts.positions[:] -= mobile_com
 
             # R acts to the left & is broadcasted N times.
-            self._ts.positions[:,:] = (self._mobile_coordinates64[:] *
-                                       self._R)
+            self._ts.positions[:, :] = self._ts.positions * self._R
```

Comment:
The backtics are [standard with numpy style](https://github.com/numpy/numpy/blob/master/doc/example.py). I think @kain88-de also mentioned the inconsistency in the docs with asterisk/backtics but [our doc style guide](https://github.com/MDAnalysis/mdanalysis/wiki/Style-Guide#writing-docstrings) does not really prescribe anything.